src/app/UrhoApp.cxx

```diff
@@ -66,18 +66,10 @@ void UrhoApp::Stop() {}
 
 void UrhoApp::create_scene() {
     auto* const cache = GetSubsystem<Urho3D::ResourceCache>();
-    m_scene = Urho3D::MakeShared<Urho3D::Scene>(context_);
-    m_scene->CreateComponent<Urho3D::Octree>();
-    m_scene->CreateComponent<EmulGlue>();
-    m_scene->CreateComponent<TorchMenu>();
-    Urho3D::Node* plane_node = m_scene->CreateChild("Ground");
-    plane_node->SetScale(Urho3D::Vector3(200, 0, 200));
-    auto* const planeObject = plane_node->CreateComponent<Urho3D::StaticModel>();
-    planeObject->SetModel(cache->GetResource<Urho3D::Model>("Models/Cone.mdl"));
-
-    Urho3D::Node* lightNode = m_scene->CreateChild("Sunlight");
-    auto* const light = lightNode->CreateComponent<Urho3D::Light>();
-    light->SetLightType(Urho3D::LIGHT_DIRECTIONAL);
+    Urho3D::ResourceCache* plane_node = GetSubsystem<Urho3D::ResourceCache>();
+    m_scene = new Urho3D::Scene(context_);
+    Urho3D::SharedPtr<Urho3D::File> file = cache->GetFile("Torch/Data/UI/smartCarEnvironment.xml");
```

Comment:
You deleted important components like TorchMenu and EmulGlue, TorchMenu adds to cache path for Torch/ without it urho woudnt be able to find the xml file (just tested it)

Reply:
that was a mistake I will add them back